create-aws-resources.sh

```diff
@@ -150,3 +145,19 @@ aws \
   --endpoint-url http://localhost:4566 \
   --rule OrderTimeoutPollerRule \
   --targets "Id"="1","Arn"="arn:aws:lambda:us-east-1:000000000000:function:orderTimeoutPoller"
+
+echo "Map Order Timeout Lambda to Order Timeout Queue"
```

Comment:
I can't tell what's changed here... can you move it back up to the order timeout section?